sdk/src/main/scala/io/horizen/account/websocket/WebSocketAccountServerEndpoint.scala

```diff
@@ -131,59 +131,66 @@ class WebSocketAccountServerEndpoint() extends SparkzLogging {
       WebSocketAccountServerEndpoint.send(new RpcResponseSuccess(rpcRequest.id, true), session)
   }
 
-  private def validateRpcRequestParams(rpcParams: JsonNode, rpcRequest: RpcId, session: Session): Unit = {
-    if (rpcParams == null || !rpcParams.isArray || rpcParams.size() < 1) {
-      log.debug("Missing or empty field params.")
-      WebSocketAccountServerEndpoint.send(new RpcResponseError(rpcRequest,
-        new RpcError(RpcCode.InvalidParams, "Missing or empty field params.", "")),
-        session)
-    }
+  private def badRpcRequestParams(rpcParams: JsonNode): Boolean = {
+    rpcParams == null || !rpcParams.isArray || rpcParams.size() < 1
   }
 
   private def createSubscriptionId(): BigInteger = {
     BigInteger.valueOf(WebSocketAccountServerEndpoint.subscriptionCounter.incrementAndGet())
   }
+
+  private def tryGetRpcRequestId(message: String): Option[RpcId] = {
+    var rpcId: Option[RpcId] = Option.empty
+    try {
+      val rpcRequest = new RpcRequest(EthJsonMapper.getMapper.readTree(message))
```

Comment:
in case someone send us `RpcRequest` partially malformed we will throw an exception in `onMessageReceived` that then will be processed by this method in the exact same way.

Instead here we need to get jsonNode `EthJsonMapper.getMapper.readTree(message)` and try get `id` field manually.

extend unit or python test with this case: for example try to send request with `id` field, but without `method`

Reply:
Done